### PR TITLE
onl: update to latest and use commit date as version

### PR DESCRIPTION
**Pull request by KanjiMonster**
There had been some recent movement at ONL, so update our checkout to latest HEAD and rebase our patches.

Since there is no usable versioning at ONL, and a constant "1.1" isn't really meaningful, switch to using the ONL commit's commit date as the version.

Run-tested on AS4630 and AS4610.

### CODE REVIEW

File: recipes-extended/onl/onl_git.bb [hunk 1]
```diff
@@ -10,6 +10,7 @@ LIC_FILES_CHECKSUM += "file://LICENSE;beginline=14;md5=7e6d108802170df125adb4f45
 
 SRCREV_onl ?= "dd61cc1792290a459c394c5b7bdf4d4a27b6f651"
 URI_ONL ?= "git://github.com/opencomputeproject/OpenNetworkLinux.git;protocol=https;branch=master"
+PV = "2022-04-08+git${SRCPV}"
```
Comment (ideaship) [inline recipes-extended/onl/onl_git.bb]: Can you add a comment above this line to let the next person to update PV know when and how to update the string?

File: recipes-extended/onl/files/accton-as5835-54x/0001-Support-psu_fan_dir-sysfs-for-YM-1401A-PSU.patch [hunk 2]
```diff
@@ -44,7 +40,10 @@ index 5964e28a..3a94da50 100755
 +                else {
 +                    data->mfr_serial[ARRAY_SIZE(data->mfr_serial)-1] = '\0';
 +                }
-+
+ 
+-            if (status < 0) {
+-                dev_dbg(&client->dev, "reg %d, err %d\n", command, status);
+-                goto exit;
```
Comment (ideaship) [inline recipes-extended/onl/files/accton-as5835-54x/0001-Support-psu_fan_dir-sysfs-for-YM-1401A-PSU.patch:46]: The commit message does not cover the change in ym2651y.c. Is this change intentional?

**Review by ideaship — CHANGES_REQUESTED** [review]

File: recipes-extended/onl/files/accton-as5835-54x/0001-Support-psu_fan_dir-sysfs-for-YM-1401A-PSU.patch (hunk 2 above)
Comment (KanjiMonster) [reply recipes-extended/onl/files/accton-as5835-54x/0001-Support-psu_fan_dir-sysfs-for-YM-1401A-PSU.patch:46]: I assume different git version decided to produce a different diff, since the effective change looks identical (instead of deleting an empty line, then adding one again, it added stuff before it and left the empty line intact).

File: recipes-extended/onl/onl_git.bb (hunk 1 above)
Comment (KanjiMonster) [reply recipes-extended/onl/onl_git.bb]: Added a comment.

**Comment by ideaship** [comment]
LGTM

**Review by ideaship — APPROVED** [review]
LGTM
